editor/styling/stylesheets/_editor.scss

```diff
@@ -201,3 +201,11 @@ Links:
 #sponsor-push > .hyperlink:hover {
   -fx-text-fill: #89d2ff;
 }
+
+.key-button {
```

Comment:
I tried put these config directly in the editor.css.
Then later, I copy the config here.

I would like to know, what is the process when build to generate css from scss.